lib/carrierwave/fog.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+CarrierWave.configure do |config|
+  config.fog_credentials = {
+    provider: 'AWS', # required
+    aws_access_key_id: 'AKIAX3XNT6YRTBCEMZUR', # required unless using use_iam_profile
```

Comment:
I think these credentials should be hidden

Reply:
Yes, you are right, but this one are useless here (old version,willdelete themat all)